pubspec.yaml

```diff
@@ -29,12 +29,14 @@ dependencies:
   flutter_bloc: ^3.1.0
   flutter_html: ^0.11.1
   flutter_local_notifications: ^0.9.1+3
+  flutter_markdown: ^0.3.2
   flutter_sound:
     git:
       url: https://github.com/open-xchange/flutter_sound
   fluttertoast: ^3.1.3
   gettext: ^1.1.0+2
   gettext_parser: ^0.1.0+1
+  html2md:
```

Comment:
We should add a specific version here.

Reply:
But they don't have any. See [project site](https://github.com/jarontai/html2md).

Reply:
For this checking pub.dev is always the right source:
https://pub.dev/packages/html2md
html2md 0.5.1